dev/benchmarks/test_apps/stocks/test/locale_test.dart

```diff
@@ -11,18 +11,10 @@ void main() {
 
   testWidgets('Changing locale', (WidgetTester tester) async {
     stocks.main();
-    await tester.idle(); // see https://github.com/flutter/flutter/issues/1865
```

Comment:
This is no longer necessary. I'm not exactly sure why, but my hunch is that its because the delegate uses a `SynchronousFuture` and is no longer really asynchronous?

Reply:
I think that's correct